x/evm/keeper/msg_server.go

```diff
@@ -812,8 +812,10 @@ func (s msgServer) VoteConfirmDeposit(c context.Context, req *types.VoteConfirmD
 	if err != nil {
 		return nil, err
 	}
+
 	event.AppendAttributes(sdk.NewAttribute(types.AttributeKeyTransferID, transferID.String()))
-	s.Logger(ctx).Debug(fmt.Sprintf("confirmed deposit for %s with transfer ID %d", depositAddr.Address, transferID))
+
+	s.Logger(ctx).Debug(fmt.Sprintf("confirmed deposit %s for %s with transfer ID %d (command ID %s)", pendingDeposit.TxID.Hex(), depositAddr.Address, transferID, types.TransferIDtoCommandID(transferID)))
```

Comment:
```suggestion
	s.Logger(ctx).Debug(fmt.Sprintf("confirmed deposit", "txID", pendingDeposit.TxID.Hex(), "address", depositAddr.Address, "transferID", transferID, "commandID", types.TransferIDtoCommandID(transferID)))
```

Reply:
Adding the extra info to the message content for now